README.md

```diff
@@ -1,96 +1,127 @@
-[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# Zendesk Support (Source)
-
-This package models Zendesk Support data from [Fivetran's connector](https://fivetran.com/docs/applications/zendesk). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/zendesk#schemainformation).
-
-This package enriches your Fivetran data by doing the following:
-
-* Adds descriptions to tables and columns that are synced using Fivetran
-* Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
-* Models staging tables, which will be used in our transform package
-
-## Models
-
-This package contains staging models, designed to work simultaneously with our [Zendesk Support transformation package](https://github.com/fivetran/dbt_zendesk). The staging models name columns consistently across all packages:
-* Boolean fields are prefixed with `is_` or `has_`
-* Timestamps are appended with `_at`
-* ID primary keys are prefixed with the name of the table. For example, the ticket table's ID column is renamed `ticket_id`.
-
-## Installation Instructions
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
-
-Include in your `packages.yml`
+<p align="center">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_zendesk_source/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="Fivetran-Release"
+        href="https://fivetran.com/docs/getting-started/core-concepts#releasephases">
+        <img src="https://img.shields.io/badge/Fivetran Release Phase-_Beta-orange.svg" /></a>
```

Comment:
```suggestion
```

We no longer want to include this badge in the README's since it is a bit confusing for customers.